src/message.rs

```diff
@@ -16,6 +16,8 @@ pub enum MessageContent {
     VoteMsg(Vote),
     /// Notification that we believe this vote to be agreed by all the listed members.
     VoteAgreedMsg((Vote, BTreeSet<Name>)),
+    /// Collection of agreed votes, sent during a merge.
+    VoteBundle(BTreeSet<(Vote, BTreeSet<Name>)>),
```

Comment:
Would it be better (especially regarding the message filter) to send a bunch of `VoteMsg`s instead of a single `VoteBundle`?